src/corehost/cli/nethost/nethost.h

```diff
@@ -65,7 +83,7 @@ extern "C" {
 NETHOST_API int NETHOST_CALLTYPE get_hostfxr_path(
     char_t * buffer,
     size_t * buffer_size,
-    const char_t * assembly_path);
+    const struct get_hostfxr_parameters *parameters);
```

Comment:
Are we okay to make this breaking change?

Reply:
No matter what version of the change we take it is breaking, so we might as well do it "right" 😄 .
Even adding a parameter is breaking - and it's the same break as changing the parameter - this is C, so no optional parameters and so on unfortunately.

Reply:
We've discussed the breaking change part of this already and it seems to be accepted. We'll need to run it again through triage though.

Reply:
I originally thought another API would be added, like `get_hostfxr_path_with_parameters`

Reply:
I see - well, if it gets rejected we can go with that, but let's see.

Reply:
Checked again with folks over here and the direction I got is that we should be fine to take the breaking change in preview 8.